lib/network/modules/components/nodes/util/NodeBase.js

```diff
@@ -43,6 +55,13 @@ class NodeBase {
     labelModule.adjustSizes(this.margin)
   }
 
+  /**
+   *
+   * @param {CanvasRenderingContext2D} ctx
+   * @param {Number} angle
+   * @returns {Number}
+   * @private
```

Comment:
`protected`?

What is the prepending of underscore used for? I understood it to be a hint that the function/property was private.